src/components/Organisms/Donate/Form/Form.js

```diff
@@ -292,7 +292,7 @@ const Signup = ({
               ref={amountRef}
             />
           </FormFieldset>
-          {amountDonate >= 1 && !noMoneyBuys && moneyBuyCopy && (
+          {amountDonate >= 1 && moneyBuyCopy && (
```

Comment:
Removes this condition so the 'Other Amount' copy will be shown regardless of the No Moneybuys setting